src/lib/cozy-client/slices/sharings.js

```diff
@@ -238,6 +238,38 @@ const createSharingLink = document => ({
   promise: client => client.createSharingLink(getPermissionsFor(document, true))
 })
 
+// TODO: this is a poor man's migration in order to normalize contacts
+// and should be removed after a few weeks in prod
```

Comment:
maybe we could add an explicit due date

Reply:
Problem is: I don't know :/ It depends on when this patch will be released to all users

Reply:
We could add

```js
// Note for future-self: If you have no idea of what it means, please, erase this code.
```

Reply:
:+1: